rust/kcl-lib/src/execution/exec_ast.rs

```diff
@@ -1957,17 +1957,11 @@ fn assign_args_to_params_kw(
                 .mut_stack()
                 .add(param.identifier.name.clone(), arg_val, (&param.identifier).into())?;
         } else {
-            // TODO: Get the actual source range.
-            // Part of https://github.com/KittyCAD/modeling-app/issues/6613
-            let pipe_value_source_range = Default::default();
-            let default_unlabeled = exec_state
-                .mod_local
-                .pipe_value
-                .clone()
-                .map(|val| Arg::new(val, pipe_value_source_range));
-            let Some(unlabeled) = args.unlabeled.take().or(default_unlabeled) else {
+            let unlabelled = args.kw_args.unlabeled.take().or_else(|| args.pipe_value.take());
```

Comment:
We already have `Args::unlabeled_kw_arg_unconverted()` that computes the unlabeled arg from all its possible inputs. To reduce duplication, or at least keep it together, could we maybe make another flavored function over there in `Args` that takes ownership, instead of reaching into it from here?